.github/workflows/ci.yml

```diff
@@ -86,8 +86,8 @@ jobs:
           path: ~/.npm
           key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
       - run: npm ci
-      - run: docker compose -f docker-compose.test.yml up
-      - run: . .env.test && npx jest --runInBand
+      - run: npm run test:services
+      - run: npm run test
```

Comment:
shouldn;t it be just npm run test, why do we have test:services?

Reply:
we set up a database for integration tests